From a3af4a405baf5ff582e82aaba392dd9667d94bdc Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 27 Aug 2018 21:24:20 +0800
Subject: [PATCH] `named/lwresd -V' and start log hide build options

The build options expose build path directories, so hide them.
[snip]
$ named -V
|built by make with *** (options are hidden)
[snip]

Upstream-Status: Inappropriate [oe-core specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Refreshed for 9.16.0
Signed-off-by: Armin Kuster <akuster@mvista.com>

---
 bin/named/include/named/globals.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: bind-9.16.0/bin/named/include/named/globals.h
===================================================================
--- bind-9.16.0.orig/bin/named/include/named/globals.h
+++ bind-9.16.0/bin/named/include/named/globals.h
@@ -69,7 +69,7 @@ EXTERN const char *named_g_version     I
 EXTERN const char *named_g_product     INIT(PRODUCT);
 EXTERN const char *named_g_description INIT(DESCRIPTION);
 EXTERN const char *named_g_srcid       INIT(SRCID);
-EXTERN const char *named_g_configargs  INIT(CONFIGARGS);
+EXTERN const char *named_g_configargs  INIT("*** (options are hidden)");
 EXTERN const char *named_g_builder     INIT(BUILDER);
 EXTERN in_port_t named_g_port	       INIT(0);
 EXTERN isc_dscp_t named_g_dscp	       INIT(-1);
